packages/deprecated-component-library-helpers/styles/type.scss

```diff
@@ -1,6 +1,8 @@
 @import "~@kaizen/design-tokens/sass/spacing";
 @import "~@kaizen/design-tokens/sass/color";
 @import "~@kaizen/design-tokens/sass/typography";
+@import "~@kaizen/deprecated-component-library-helpers/styles/color";
+@import "~@kaizen/deprecated-component-library-helpers/styles/layers";
```

Comment:
These are the two imports that are in `packages/component-library/styles/type.scss` but not in `packages/deprecated-component-library-helpers/styles/type.scss`